src/components/Markdown/Tiles.tsx

```diff
@@ -0,0 +1,36 @@
+import Link from 'src/components/Link';
+import Icon from '@ably/ui/core/Icon';
+import { IconName } from '@ably/ui/core/Icon/types';
+
+export type TileProps = {
+  title: string;
+  description: string;
+  image: IconName;
+  link: string;
+};
+
+export const Tile = ({ title, description, image, link }: TileProps) => {
+  return (
+    <Link to={link} rel="noreferrer" className="block no-underline">
```

Comment:
These links are all to internal stuff right? In which case we don't need `rel="noreferrer"`